src/abi/method.ts

```diff
@@ -160,3 +160,20 @@ export class ABIMethod {
     });
   }
 }
+
+export function getMethodByName(methods: ABIMethod[], name: string): ABIMethod {
```

Comment:
I think it would be nice if there was a guard statement that checks `methods` is not `null` or is `instance of ABIMethod[]` here

Reply:
added this

Reply:
nice, thanks! Looks good to me